internal/endtoend/testdata/emit_result_and_params_struct_pointers/go/query.sql.go

```diff
@@ -0,0 +1,74 @@
+// Code generated by sqlc. DO NOT EDIT.
+// source: query.sql
+
+package querytest
+
+import (
+	"context"
+	"database/sql"
+)
+
+const getAll = `-- name: GetAll :many
+SELECT a, b FROM foo
+`
+
+func (q *Queries) GetAll(ctx context.Context) ([]*Foo, error) {
```

Comment:
The slice of structs seems weird to me, but I guess that's what you'd want in this case?

Reply:
There has been some discussion about performance https://stackoverflow.com/questions/27622083/slices-of-structs-vs-slices-of-pointers-to-structs